ogc/tiles/main_test.go

```diff
@@ -171,8 +171,8 @@ func TestTiles_Tile(t *testing.T) {
 				tileCol:         "15",
 			},
 			want: want{
```

Comment:
Dekken we nu alle scenario's af? Ik voorzien de volgende:

- Accept header: <mapbox vector tiles mediatype>
- ?f=mvt
- ?f=pbf
- .pbf (file extensie, zoals gebruit in v0.1 vector tiles)

Reply:
Zou wel mooi zijn om daar unit testen voor te hebben

Reply:
Volgens mij zijn alle mogelijke scenario's de vier die je noemt, en nu dus de default (als je geen `f` param en geen header opgeeft, geeft 'ie toch mvt terug). Fout scenario is dan als je expliciet een fout format opgeeft (e.g., `?f=tilejson`).
Ik voeg nog even tests toe voor `?f=pbf` en `?f=tilejson`